src/openrct2/ride/ShopItem.h

```diff
@@ -94,6 +94,7 @@ struct rct_shop_item_string_types
 
 extern uint32 gSamePriceThroughoutParkA;
 extern uint32 gSamePriceThroughoutParkB;
+struct Ride;
```

Comment:
Personal preference, but I think forward declarations would fit better right after the include list.

Reply:
Updated